pkgs/by-name/ap/apple-sdk/common/rewrite-sdk-paths.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, sdkVersion }:
+
+let
+  name = "MacOSX${lib.versions.majorMinor sdkVersion}.sdk";
+in
+self: super: {
+  # Rewrite the stubs to point to dylibs in the SDK instead of at system locations. This is needed for umberalla
+  # frameworks in older SDKs, which don’t also embed their stubs.
```

Comment:
Maybe, but I haven’t checked whether there are frameworks other than libSystem that only include references to sub-frameworks instead of embedding their stubs in the umbrella definition.